tests/beaker_tests/cisco_bgp/bgplib.rb

```diff
@@ -237,10 +234,28 @@ def self.create_bgp_manifest_present_non_default
       }
     }
     EOF"
+
     manifest_str
   end
   # rubocop:enable Metrics/MethodLength
 
+  # creat manifest for difference on platform
```

Comment:
Can this be expanded on for clarity? What difference(s), what platform(s), why is this needed?